### PR TITLE
User research 20160426

### PR DESCRIPTION
**Pull request by madgoose**
https://www.pivotaltracker.com/story/show/117520807

### CODE REVIEW

**Comment by ibrechin** [comment]
@madgoose those commits can probably be squashed

File: mtp_send_money/assets-src/stylesheets/app.scss [hunk 1]
```diff
@@ -70,6 +70,15 @@ form .form-group .form-hint {
   margin-top: 2em;
 }
 
+.mtp-bank-details {
+    background: #fff;
```
Comment (maxf) [inline mtp_send_money/assets-src/stylesheets/app.scss]: Please use $page-colour

File: mtp_send_money/assets-src/stylesheets/app.scss [hunk 2]
```diff
@@ -70,6 +70,15 @@ form .form-group .form-hint {
...
+    border: #808080 solid 1px;
+    padding: 5px 20px;
```
Comment (maxf) [inline mtp_send_money/assets-src/stylesheets/app.scss]: Same here: always safer to use colour variables from http://govuk-elements.herokuapp.com/colour/

File: mtp_send_money/assets-src/stylesheets/app.scss [hunk 3]
```diff
@@ -70,6 +70,15 @@ form .form-group .form-hint {
...
+    font-size: 24px;
+    font-weight: bold;
```
Comment (maxf) [inline mtp_send_money/assets-src/stylesheets/app.scss]: em preferred, if possible

File: mtp_send_money/templates/send_money/bank-transfer.html [hunk 4]
```diff
@@ -7,28 +7,34 @@
...
+      <samp class="mtp-bank-details">{{ account_number }}</samp>
+    </dd>
+    <dt>{% trans 'Use this reference' %}</dt>
+    <dd>
+      <samp class="mtp-bank-details">{{ bank_transfer_reference }}</samp>
+    </dd>
+  </dl>
+  <p><strong>{% trans 'For the payment to work it is essential you use the reference we have generated for you.' %}</strong></p>
+  
+  <h3>{% trans 'When will the prisoner receive the money?' %}</h3>
```
Comment (maxf) [inline mtp_send_money/templates/send_money/bank-transfer.html]: You could change still whole section to be under one blocktrans

Comment (madgoose) [inline mtp_send_money/templates/send_money/bank-transfer.html]: duly noted! bear in mind this just a prototype though…

File: mtp_send_money/assets-src/stylesheets/app.scss (hunk 3 above)
Comment (madgoose) [inline mtp_send_money/assets-src/stylesheets/app.scss]: duly noted! i just copied the (compiled) styles from http://sendmoneytoaprisoner.service.justice.gov.uk/

File: mtp_send_money/assets-src/stylesheets/app.scss (hunk 2 above)
Comment (madgoose) [inline mtp_send_money/assets-src/stylesheets/app.scss]: a good point. i just copied the (compiled) styles from http://sendmoneytoaprisoner.service.justice.gov.uk/

**Pull request merged**